PubnativeLite/PubnativeLite/Ad Model/HyBidAd.m

```diff
@@ -96,6 +96,32 @@ - (instancetype)initWithVWXml:(NSDictionary *)xml andWithAdSize:(HyBidAdSize *)a
     return self;
 }
 
+- (instancetype)initWithVWVASTXml:(NSDictionary *)xml andWithAdSize:(HyBidAdSize *)adSize {
+    self = [super init];
+    if (self) {
+        HyBidAdModel *model = [[HyBidAdModel alloc] init];
+        self.adSize = adSize;
+        NSString *apiAsset = PNLiteAsset.vast;
+        NSMutableArray *assets = [[NSMutableArray alloc] init];
+        NSMutableString *xmlString = [[NSMutableString alloc] init];
+        for (NSString *key in [xml allKeys]) {
```

Comment:
Check the comment below. If you pass to the constructor the NSData as a string you don't need to convert the dictionary back to XML.

Reply:
Made the changes. You can check my latest commit.